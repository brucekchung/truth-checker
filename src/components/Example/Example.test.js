/*eslint-disable*/
import React from 'react'
import { shallow } from 'enzyme'
import { Example } from './Example'

describe('Example', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<Example />)

    expect(wrapper).toMatchSnapshot()
  })
})