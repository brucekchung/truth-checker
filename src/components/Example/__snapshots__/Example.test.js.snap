// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Example should match snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Unknown />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h2>
          Examples 
</h2>,
        <h4>
          Copy and paste any of the following links into the search bar to see how News-rater responds to a variety of sources:
</h4>,
        <div
          className="card"
>
          <h5>
                    Features tone of bias against a political figure
          </h5>
          <p>
                    https://www.bloomberg.com/news/articles/2018-01-09/trump-loving-farmers-let-him-know-that-they-also-love-free-trade
          </p>
</div>,
        <div
          className="card"
>
          <h5>
                    Informative article about travel
          </h5>
          <p>
                    https://www.cnn.com/travel/article/passenger-planes-future-look/index.html
          </p>
</div>,
        <div
          className="card"
>
          <h5>
                    Satirical article from The Onion
          </h5>
          <p>
                    https://www.theonion.com/nra-calls-for-teachers-to-keep-loaded-gun-pointed-at-cl-1819575763
          </p>
</div>,
        <div
          className="card"
>
          <h5>
                    Article from a polarizing site (Brietbart News) containing a somewhat racy view
          </h5>
          <p>
                    http://www.breitbart.com/big-journalism/2018/02/27/jorge-ramos-very-sad-latino-trump-voters-turned-backs-on-immigrants/
          </p>
</div>,
        <div
          className="card"
>
          <h5>
                    Article from a polarizing site (Brietbart News) on school shootings
          </h5>
          <p>
                    http://www.breitbart.com/big-government/2018/02/27/poll-americans-blame-school-shooting-government-not-guns/
          </p>
</div>,
      ],
      "className": "Example",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Examples ",
        },
        "ref": null,
        "rendered": "Examples ",
        "type": "h2",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Copy and paste any of the following links into the search bar to see how News-rater responds to a variety of sources:",
        },
        "ref": null,
        "rendered": "Copy and paste any of the following links into the search bar to see how News-rater responds to a variety of sources:",
        "type": "h4",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <h5>
              Features tone of bias against a political figure
</h5>,
            <p>
              https://www.bloomberg.com/news/articles/2018-01-09/trump-loving-farmers-let-him-know-that-they-also-love-free-trade
</p>,
          ],
          "className": "card",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Features tone of bias against a political figure",
            },
            "ref": null,
            "rendered": "Features tone of bias against a political figure",
            "type": "h5",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "https://www.bloomberg.com/news/articles/2018-01-09/trump-loving-farmers-let-him-know-that-they-also-love-free-trade",
            },
            "ref": null,
            "rendered": "https://www.bloomberg.com/news/articles/2018-01-09/trump-loving-farmers-let-him-know-that-they-also-love-free-trade",
            "type": "p",
          },
        ],
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <h5>
              Informative article about travel
</h5>,
            <p>
              https://www.cnn.com/travel/article/passenger-planes-future-look/index.html
</p>,
          ],
          "className": "card",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Informative article about travel",
            },
            "ref": null,
            "rendered": "Informative article about travel",
            "type": "h5",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "https://www.cnn.com/travel/article/passenger-planes-future-look/index.html",
            },
            "ref": null,
            "rendered": "https://www.cnn.com/travel/article/passenger-planes-future-look/index.html",
            "type": "p",
          },
        ],
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <h5>
              Satirical article from The Onion
</h5>,
            <p>
              https://www.theonion.com/nra-calls-for-teachers-to-keep-loaded-gun-pointed-at-cl-1819575763
</p>,
          ],
          "className": "card",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Satirical article from The Onion",
            },
            "ref": null,
            "rendered": "Satirical article from The Onion",
            "type": "h5",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "https://www.theonion.com/nra-calls-for-teachers-to-keep-loaded-gun-pointed-at-cl-1819575763",
            },
            "ref": null,
            "rendered": "https://www.theonion.com/nra-calls-for-teachers-to-keep-loaded-gun-pointed-at-cl-1819575763",
            "type": "p",
          },
        ],
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <h5>
              Article from a polarizing site (Brietbart News) containing a somewhat racy view
</h5>,
            <p>
              http://www.breitbart.com/big-journalism/2018/02/27/jorge-ramos-very-sad-latino-trump-voters-turned-backs-on-immigrants/
</p>,
          ],
          "className": "card",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Article from a polarizing site (Brietbart News) containing a somewhat racy view",
            },
            "ref": null,
            "rendered": "Article from a polarizing site (Brietbart News) containing a somewhat racy view",
            "type": "h5",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "http://www.breitbart.com/big-journalism/2018/02/27/jorge-ramos-very-sad-latino-trump-voters-turned-backs-on-immigrants/",
            },
            "ref": null,
            "rendered": "http://www.breitbart.com/big-journalism/2018/02/27/jorge-ramos-very-sad-latino-trump-voters-turned-backs-on-immigrants/",
            "type": "p",
          },
        ],
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <h5>
              Article from a polarizing site (Brietbart News) on school shootings
</h5>,
            <p>
              http://www.breitbart.com/big-government/2018/02/27/poll-americans-blame-school-shooting-government-not-guns/
</p>,
          ],
          "className": "card",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Article from a polarizing site (Brietbart News) on school shootings",
            },
            "ref": null,
            "rendered": "Article from a polarizing site (Brietbart News) on school shootings",
            "type": "h5",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "http://www.breitbart.com/big-government/2018/02/27/poll-americans-blame-school-shooting-government-not-guns/",
            },
            "ref": null,
            "rendered": "http://www.breitbart.com/big-government/2018/02/27/poll-americans-blame-school-shooting-government-not-guns/",
            "type": "p",
          },
        ],
        "type": "div",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h2>
            Examples 
</h2>,
          <h4>
            Copy and paste any of the following links into the search bar to see how News-rater responds to a variety of sources:
</h4>,
          <div
            className="card"
>
            <h5>
                        Features tone of bias against a political figure
            </h5>
            <p>
                        https://www.bloomberg.com/news/articles/2018-01-09/trump-loving-farmers-let-him-know-that-they-also-love-free-trade
            </p>
</div>,
          <div
            className="card"
>
            <h5>
                        Informative article about travel
            </h5>
            <p>
                        https://www.cnn.com/travel/article/passenger-planes-future-look/index.html
            </p>
</div>,
          <div
            className="card"
>
            <h5>
                        Satirical article from The Onion
            </h5>
            <p>
                        https://www.theonion.com/nra-calls-for-teachers-to-keep-loaded-gun-pointed-at-cl-1819575763
            </p>
</div>,
          <div
            className="card"
>
            <h5>
                        Article from a polarizing site (Brietbart News) containing a somewhat racy view
            </h5>
            <p>
                        http://www.breitbart.com/big-journalism/2018/02/27/jorge-ramos-very-sad-latino-trump-voters-turned-backs-on-immigrants/
            </p>
</div>,
          <div
            className="card"
>
            <h5>
                        Article from a polarizing site (Brietbart News) on school shootings
            </h5>
            <p>
                        http://www.breitbart.com/big-government/2018/02/27/poll-americans-blame-school-shooting-government-not-guns/
            </p>
</div>,
        ],
        "className": "Example",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Examples ",
          },
          "ref": null,
          "rendered": "Examples ",
          "type": "h2",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Copy and paste any of the following links into the search bar to see how News-rater responds to a variety of sources:",
          },
          "ref": null,
          "rendered": "Copy and paste any of the following links into the search bar to see how News-rater responds to a variety of sources:",
          "type": "h4",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <h5>
                Features tone of bias against a political figure
</h5>,
              <p>
                https://www.bloomberg.com/news/articles/2018-01-09/trump-loving-farmers-let-him-know-that-they-also-love-free-trade
</p>,
            ],
            "className": "card",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Features tone of bias against a political figure",
              },
              "ref": null,
              "rendered": "Features tone of bias against a political figure",
              "type": "h5",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "https://www.bloomberg.com/news/articles/2018-01-09/trump-loving-farmers-let-him-know-that-they-also-love-free-trade",
              },
              "ref": null,
              "rendered": "https://www.bloomberg.com/news/articles/2018-01-09/trump-loving-farmers-let-him-know-that-they-also-love-free-trade",
              "type": "p",
            },
          ],
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <h5>
                Informative article about travel
</h5>,
              <p>
                https://www.cnn.com/travel/article/passenger-planes-future-look/index.html
</p>,
            ],
            "className": "card",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Informative article about travel",
              },
              "ref": null,
              "rendered": "Informative article about travel",
              "type": "h5",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "https://www.cnn.com/travel/article/passenger-planes-future-look/index.html",
              },
              "ref": null,
              "rendered": "https://www.cnn.com/travel/article/passenger-planes-future-look/index.html",
              "type": "p",
            },
          ],
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <h5>
                Satirical article from The Onion
</h5>,
              <p>
                https://www.theonion.com/nra-calls-for-teachers-to-keep-loaded-gun-pointed-at-cl-1819575763
</p>,
            ],
            "className": "card",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Satirical article from The Onion",
              },
              "ref": null,
              "rendered": "Satirical article from The Onion",
              "type": "h5",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "https://www.theonion.com/nra-calls-for-teachers-to-keep-loaded-gun-pointed-at-cl-1819575763",
              },
              "ref": null,
              "rendered": "https://www.theonion.com/nra-calls-for-teachers-to-keep-loaded-gun-pointed-at-cl-1819575763",
              "type": "p",
            },
          ],
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <h5>
                Article from a polarizing site (Brietbart News) containing a somewhat racy view
</h5>,
              <p>
                http://www.breitbart.com/big-journalism/2018/02/27/jorge-ramos-very-sad-latino-trump-voters-turned-backs-on-immigrants/
</p>,
            ],
            "className": "card",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Article from a polarizing site (Brietbart News) containing a somewhat racy view",
              },
              "ref": null,
              "rendered": "Article from a polarizing site (Brietbart News) containing a somewhat racy view",
              "type": "h5",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "http://www.breitbart.com/big-journalism/2018/02/27/jorge-ramos-very-sad-latino-trump-voters-turned-backs-on-immigrants/",
              },
              "ref": null,
              "rendered": "http://www.breitbart.com/big-journalism/2018/02/27/jorge-ramos-very-sad-latino-trump-voters-turned-backs-on-immigrants/",
              "type": "p",
            },
          ],
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <h5>
                Article from a polarizing site (Brietbart News) on school shootings
</h5>,
              <p>
                http://www.breitbart.com/big-government/2018/02/27/poll-americans-blame-school-shooting-government-not-guns/
</p>,
            ],
            "className": "card",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Article from a polarizing site (Brietbart News) on school shootings",
              },
              "ref": null,
              "rendered": "Article from a polarizing site (Brietbart News) on school shootings",
              "type": "h5",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "http://www.breitbart.com/big-government/2018/02/27/poll-americans-blame-school-shooting-government-not-guns/",
              },
              "ref": null,
              "rendered": "http://www.breitbart.com/big-government/2018/02/27/poll-americans-blame-school-shooting-government-not-guns/",
              "type": "p",
            },
          ],
          "type": "div",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
